### PR TITLE
ENCD-6169 gkm-SVM-model

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

**Review by bonitalam — CHANGES_REQUESTED** [review]
Looks good to me, just missing updates in the file changelog

File: src/encoded/schemas/changelogs/file.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 * Added *origin_batches* calculated property
 * Added *regulatory elements prediction model* to the enum list for *output_type*
 * Added *genome subcompartments* and *chromatin stripes* to the enum list for *output_type*
+* Added *TF binding prediction model*, *enhancer prediction model* and *"*promoter prediction model*  to the enum list for *output_type*, restricted the use of *regulatory elements prediction model* to DCC personell only
```
Comment (bonitalam) [inline src/encoded/schemas/changelogs/file.md]: Just one typo - `personell` should be `personnel`

Comment (bonitalam) [reply src/encoded/schemas/changelogs/file.md]: Also you have a double quote and extra asterisk in `*"*promoter prediction model*` that should be removed to be `*promoter prediction model*`

Comment (gabdank) [reply src/encoded/schemas/changelogs/file.md]: fixed

**Review by bonitalam — DISMISSED** [review]

File: src/encoded/schemas/changelogs/file.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 * Added *origin_batches* calculated property
 * Added *regulatory elements prediction model* to the enum list for *output_type*
 * Added *genome subcompartments* and *chromatin stripes* to the enum list for *output_type*
+* Added *TF binding prediction model*, *enhancer prediction model* and *promoter prediction model*  to the enum list for *output_type*, restricted the use of *regulatory elements prediction model* to DCC personel only
```
Comment (bonitalam) [inline src/encoded/schemas/changelogs/file.md]: sorry last typo issue @gabdank can you update to say `restricted the use of *regulatory elements prediction model* *output_type* to DCC personnel only`

Comment (bonitalam) [reply src/encoded/schemas/changelogs/file.md]: Personnel with two "n"s, thank you!

**Review by bonitalam — DISMISSED** [review]

**Review by bonitalam — APPROVED** [review]

**Pull request merged**